cli/src/file_utils.rs

```diff
@@ -9,6 +9,7 @@ use walkdir::WalkDir;
 
 static FILE_EXTENSIONS_PER_LANGUAGE_LIST: &[(Language, &[&str])] = &[
     (Language::JavaScript, &["js", "jsx"]),
+    (Language::Dockerfile, &["docker", "dockerfile"]),
```

Comment:
```suggestion
```

You don't mention this in the description nor test for it. See comments on Slack